tensorflow/security/fuzzing/cc/text_literal_reader_fuzz.cc

```diff
@@ -43,5 +44,13 @@ void FuzzFileRead(std::string data) {
 }
 FUZZ_TEST(TextReaderFuzzer, FuzzFileRead);
 
+void FuzzHLOParseUnverified(std::string data) {
+  auto result = xla::ParseAndReturnUnverifiedModule(data);
+  if (OkStatus() == result.status()) {
```

Comment:
This `if` seems superfluous, did you mean to error out in some of these? (same comment for the other function's ifs)

Reply:
It was an ugly hack -- I substituted for `IgnoreError` now